php/tests/generated_class_test.php

```diff
@@ -877,4 +877,279 @@ public function testPrefixForReservedWords()
         $m = new \Foo\PBEmpty();
         $m = new \PrefixEmpty();
     }
+
+    #########################################################
+    # Test phpdoc for getters and setters.
+    #########################################################
+
+    public function testPhpDocForClass()
+    {
+        $class = new ReflectionClass('Foo\TestMessage');
+        $doc = $class->getDocComment();
+        $this->assertContains('Foo&#92;TestMessage', $doc);
```

Comment:
@teboring I see these escaped characters all over the phpdocs. I do not think we need them. If this is for HTML escaping, this should be done wherever the HTML is generated. Thoughts on removing it?